NEWS

```diff
@@ -1,3 +1,11 @@
+Flocker 1.3.0 (2015-09-04)
+=====================================
+
+- New Flocker API endpoints to create, list, renew, and delete lease on a dataset.
+- Flocker now works on servers where SELinux is enabled. (FLOC-2417)
```

Comment:
I think this was a 1.2.0 feature which never got added to the NEWS file. 
See https://github.com/ClusterHQ/flocker/pull/1847/files#r37109546